zetaclient/orchestrator/bootstrap.go

```diff
@@ -363,7 +360,7 @@ func syncObserverMap(
 				continue
 			}
 
-			database, err := db.NewFromSqlite(dbpath, btcDatabaseFilename, true)
+			database, err := db.NewFromSqlite(dbpath, btcDatabaseFileName(*rawChain), true)
```

Comment:
Can we check for nil pointer dereference in the function?

Reply:
This was checked in the `zctx.Chain` [constructor](https://github.com/zeta-chain/node/blob/fix-signet-database-name/zetaclient/context/chain.go#L187) and already dereferenced [above this line](https://github.com/zeta-chain/node/blob/fix-signet-database-name/zetaclient/orchestrator/bootstrap.go#L300)